Update doc for history and change section (#192)
cisco/mindmeld: source/versions/history.rst

Package History
===============

4.3.1 (2020-06-17)
------------------

Major Features and Improvements
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> - Custom Action provides the ability for external applications to integrate custom dialogue management logic with MindMeld applications

Reviewer: ```suggestion
- Custom Actions provide the ability for applications to integrate external dialogue management logic with MindMeld applications
```

> - Question Answerer can leverage semantic embeddings (Elasticsearch 7 is required)

Reviewer: ```suggestion
- Question Answerer can now leverage deep-learning based semantic embeddings to produce more relevant answers to queries
```

Reviewer: I dont think we need to specify ES 7 here.

Author: It is specific to ES7+ right?

> - Automatic slot filing provides a useful functionality for automatically prompting the user for missing entities or slots required to fulfill an intent

Reviewer: ```suggestion
- Automatic slot filing allows an intuitive way for developers to automatically prompt users for missing slots to fulfill an intent
```

- A new banking blueprint for enterprise use-cases

> - WhatsApp Integration

Reviewer: ```suggestion
- WhatsApp Bot Integration with MindMeld
```

Package History
===============

4.3.1 (2020-06-17)
------------------

Major Features and Improvements
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- Custom Actions provide the ability for applications to integrate external dialogue management logic with MindMeld applications

- Question Answerer can now leverage deep-learning based semantic embeddings (BERT, Glove) to produce more relevant answers to queries (available for Elasticsearch 7 and above)

- Automatic slot filing allows an intuitive way for developers to automatically prompt users for missing slots to fulfill an intent

- A new banking blueprint for enterprise use-cases

- WhatsApp Bot Integration with MindMeld

- Docker setup update to Elasticsearch 7

> - MindMeld application can configure language and locale setting in the application config file

Reviewer: ```suggestion
- MindMeld application can configure language and locale settings in the application config file
```

Bug fixes
^^^^^^^^^

> - Addressed an issue which caused MindMeld to be unable to detect system entities without any context (for example: "december 21st")

Reviewer: ```suggestion
- Addressed an issue which caused MindMeld to not detect system entities with no surrounding context (for example: "december 21st")
```

> - Previously, MindMeld applications required Elasticsearch even if the application did not need it (i.e having no entity to resolve); read :doc:`Managing Dependencies <../userguide/optional_dependency>` for more details

Reviewer: ```suggestion
- Previously, MindMeld applications called Elasticsearch even if the application did not functionally use it (i.e have no entity to resolve); This has been fixed using read :doc:`Managing Dependencies <../userguide/optional_dependency>`.
```
